components/edit-collective/sections/virtual-cards/VirtualCards.js

```diff
@@ -170,7 +185,16 @@ const VirtualCards = props => {
       </Box>
       <Grid mt={4} gridTemplateColumns={['100%', '366px 366px']} gridGap="32px 24px">
         {data.account.virtualCards.nodes.map(vc => (
```

Comment:
I know it was here before but I don't know what is `vc`, "venture capitalist?

Characters are cheap, abbreviations are not useful. -> `virtualCard`.